e2e/support/helpers/e2e-embedding-iframe-sdk-helpers.ts

```diff
@@ -187,21 +187,19 @@ const convertPropertiesToEmbedTagAttributes = (
  * @param {EnabledAuthMethods[]} enabledAuthMethods - The authentication methods to enable.
  */
 export function prepareSdkIframeEmbedTest({
-  withTokenFeatures = true,
+  withToken = "bleeding-edge",
   enabledAuthMethods = ["jwt"],
   signOut = false,
 }: {
-  withTokenFeatures?: boolean;
+  withToken?: false | "starter" | "bleeding-edge";
```

Comment:
Oof. Maybe `"starter" | "bleeding-edge" | undefined` would be cleaner?

Reply:
I don't think that will work, does it? We have a default value for `withToken = "bleeding-edge"` so if you pass `undefined` it basically means you use this token. It's the current default value since we don't pass this prop in a lot of tests.